rr_common/src/follower/follower.h

```diff
@@ -8,8 +8,10 @@
 #include <ros/ros.h>
 #include <rr_msgs/speed.h>
 #include <rr_msgs/steering.h>
+#include <std_msgs/Float64.h>
 #include <string>
 #include "flann/flann.hpp"
+//#include <asdl.h>
```

Comment:
remove unused includes. asdl is commented out already and I'm pretty sure this node doesn't use FLANN (I know you didn't add that but it shouldn't have been in there to start with)